src/docfx/build/xref/XrefResolver.cs

```diff
@@ -167,9 +176,22 @@ public XrefMapModel ToXrefMapModel(bool isLocalizedBuild)
                     })
                     .OrderBy(xref => xref.Uid)
                     .ToArray();
+
+                externalXrefs = _valdiateExternalXrefs.Value.GroupBy(uid => uid.Value).Select(uidGroup =>
```

Comment:
consider delegate the whole method to _JsonSchemaTransformer_
```suggestion
                externalXrefs = _jsonSchemaTransformer.Value.GetExternalXrefs();
```

Reply:
OK.